incubator/quarkus/README.md

```diff
@@ -12,7 +12,44 @@ See: https://quarkus.io/
 
 ## Templates
 
-Templates are used to create your local project and start your development. When initializing your project you will be provided with the default template project. This template provides a simple Java application with a JAX-RS "Hello World!" REST API example and a simple HTML welcome page.
+Templates are used to create your local project and start your development. This stack provides two templates you can choose from: `default` and `kafka`. If you do not specify a template, the `default` template will be used.
+
+### `default` template
+
+This template provides a simple Java application with a JAX-RS "Hello World!" REST API example and a simple HTML welcome page.
+
+### `kafka` template
+
+This template provides a Java application which demonstrates using MicroProfile Reactive Messaging to consume and produce messages on Kafka topics. The application uses the example from the [Quarkus Kafka guide](https://quarkus.io/guides/kafka).
+
+A bean produces a random `Integer` onto a Reactive Messaging channel named "generated-price" every 5 seconds. This channel is connected to a Kafka topic named "prices". Another bean consumes from that topic, applies a conversion, and publishes a `double` onto another channel named "my-data-stream". Finally, that channel is streamed to a JAX-RS endpoint `/prices.html`, using [Server-sent Events](https://en.wikipedia.org/wiki/Server-sent_events).
+
+#### Running locally
+
+The template provides a sample `docker-compose.yaml` which you can use to run a single-node Kafka cluster on your local machine. To start Kafka locally, run `docker-compose up`. This will start Kafka, Zookeeper, and also create a Docker network on your machine, which you can find the name of by running `docker network list`.
+
+To run the application using Appsody, use this command, substituting in the name of your Docker network:
+
+`$ appsody run --network network_name --docker-options "--env KAFKA_BOOTSTRAP_SERVERS=kafka:9092"`
```

Comment:
I can reach the /prices endpoint but I get a `404 - Resource Not Found` on the default `localhost:8080` endpoint. Is that expected?

Reply:
Yes that's correct.